data.sql

```diff
@@ -4,3 +4,13 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, TRUE, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, TRUE, 11);
+INSERT INTO
+  animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  ('Charmander', '2020-02-08', 0, FALSE, -11),
+  ('Plantmon', '2021-11-15', 2, TRUE, -5.7),
+  ('Squirtle', '1993-04-02', 3, FALSE, -12.13),
+  ('Angemon', '2005-06-12', 1, TRUE, -45),
```

Comment:
- Please, kidly add a  `dot` and zero to the numbers as they're floating numbers to meet the project requirement.
`-11.0`, ` -5.7`, `-12.13` and `-45.0`

Reply:
@zieeco Thanks for the observation. I already added them. 👍